Harvest.Core/Domain/Expense.cs

```diff
@@ -59,6 +59,9 @@ public class Expense
         [StringLength(50)]
         public string Account { get; set; }
 
+        //This is actually required, but that will make the migration a pain...
+        public string ExpenseObjectCode { get; set; }
```

Comment:
After generating the migration, it is okay to manually edit the corresponding code to first create the column as nullable, update with a default value, then change it to not nullable. But yeah, it is annoying.

Reply:
oops. Just noticed it's just a draft.

Reply:
The update would be a pain. This will need to wipe the DB for proper values first.
And I'll just make sure this is set with code.